include/rocksdb/listener.h

```diff
@@ -118,6 +118,7 @@ enum class FlushReason : int {
   // When set the flush reason to kErrorRecoveryRetryFlush, SwitchMemtable
   // will not be called to avoid many small immutable memtables.
   kErrorRecoveryRetryFlush = 0xc,
+  kWALFull = 0xd,
```

Comment:
Google C++ style calls for `kWalFull`

Reply:
I spelled it consistently with other names in the file, for example

https://github.com/facebook/rocksdb/blob/a951b38e9de962eb9e9a41bdedbbe8a8743c6519/include/rocksdb/listener.h#L134

But I can change it if there's a preference.

Reply:
Yes, perhaps the most egregious violator is class `DB`. I think we're trying to follow the Google style on new code though.

    $ git grep '[^a-zA-Z]kWal' | wc -l
    78
    $ git grep '[^a-zA-Z]kWAL' | wc -l
    17